docs/indexer/quickstart/how-it-works.md

```diff
@@ -0,0 +1,160 @@
+
+1. [The project.ts file](#the-projectts-file)
+2. [The GraphQL Schema file](#the-graphql-schema-file)
+3. [The Mapping functions (in src/mappings/ directory)](#the-mapping-functions)
+
+## The project.ts file
+
+The project.ts file can be seen as an entry point of your project as it defines most of the details on how SubQuery will index and transform the chain data. It indicates where we are indexing data from, and to what on chain events we are subscribing to.
+
+This file imports various libraries and configurations along with defining the name, version and description of the project. In the network section, it specifies the chainID, the RPC endpoint, and also the startBlock. It also specifies the ABI contract address.
+
+:::details project.ts snippet
+```
+.
+.
+.
+ specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "ethereum-starter",
+  description:
+    "This project can be use as a starting point for developing your new Ethereum SubQuery project",
+  runner: {
+    node: {
+      name: "@subql/node-ethereum",
+      version: ">=3.0.0",
+    },
+    query: {
+      name: "@subql/query",
+      version: "*",
+    },
+  },
+  schema: {
+    file: "./schema.graphql",
+  },
+  network: {
+    /**
+     * chainId is the EVM Chain ID, for Ethereum this is 1
+     * https://chainlist.org/chain/1
+     */
+    chainId: process.env.CHAIN_ID!,
+    /**
+     * These endpoint(s) should be public non-pruned archive node
+     * We recommend providing more than one endpoint for improved reliability, performance, and uptime
+     * Public nodes may be rate limited, which can affect indexing speed
+     * When developing your project we suggest getting a private API key
+     * If you use a rate limited endpoint, adjust the --batch-size and --workers parameters
+     * These settings can be found in your docker-compose.yaml, they will slow indexing but prevent your project being rate limited
+     */
+    endpoint: process.env.ENDPOINT!?.split(',') as string[] | string,
+  },
+  dataSources: [
+    {
+      kind: EthereumDatasourceKind.Runtime,
+      startBlock: 4719568,
+
+      options: {
+        // Must be a key of assets
+        abi: "erc20",
+        // # this is the contract address for wrapped ether https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+        address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+      },
+      .
+      .
+      .
+```
+:::
+
+Finally, there are three types of mapping handlers (you can have more than one in each project):
+
+* BlockHandlers: On each and every block, run a mapping function
+* TransactionHandlers: On each and every transaction that matches optional filter criteria, run a mapping function
+* LogHandlers: On each and every log that matches optional filter criteria, run a mapping function
+
+The hello-world project has a transaction handler called "handleTransaction" and a LogHandler called "handleLog". When a filter condition is met, the associated handler is called. 
```

Comment:
I hope i got the mapping handlers link right here
```suggestion
The hello-world project has a transaction handler called "handleTransaction" and a LogHandler called "handleLog". These are exported functions that will be explained in the [mapping handlers](#mapping-handlers) section. When a filter condition is met, the associated handler is called. 
```

Reply:
Updated.